toqito/helper/bell_notation_conversions.py

```diff
@@ -0,0 +1,748 @@
+"""Conversions between Bell inequality notations."""
+
+import numpy as np
+
+
+def cg_to_fc(cg_mat: np.ndarray, behaviour: bool = False) -> np.ndarray:
+    r"""Convert a Bell functional or behaviour from Collins-Gisin (CG) to Full Correlator (FC) notation.
+
+    The Collins-Gisin (CG) notation for a Bell functional or behaviour is represented by a matrix:
+
+    .. math::
+        \text{CG} =
+        \begin{pmatrix}
+            K      & p_B(0|1) & p_B(0|2) & \dots \\
+            p_A(0|1) & p(00|11) & p(00|12) & \dots \\
+            p_A(0|2) & p(00|21) & p(00|22) & \dots \\
+            \vdots   & \vdots   & \vdots   & \ddots
+        \end{pmatrix}
+
+    The Full Correlator (FC) notation is represented by:
+
+    .. math::
+        \text{FC} =
+        \begin{pmatrix}
+            K      & \langle B_1 \rangle & \langle B_2 \rangle & \dots \\
+            \langle A_1 \rangle & \langle A_1 B_1 \rangle & \langle A_1 B_2 \rangle & \dots \\
+            \langle A_2 \rangle & \langle A_2 B_1 \rangle & \langle A_2 B_2 \rangle & \dots \\
+            \vdots   & \vdots      & \vdots      & \ddots
+        \end{pmatrix}
+
+    This function converts between these two notations.
+
+    Examples
+    ==========
+
+    Consider the CHSH inequality in CG notation for a functional:
+
+    .. math::
+        \text{CHSH}_{CG} =
+        \begin{pmatrix}
+            0 & 0 & 0 \\
+            0 & 1 & -1 \\
+            0 & -1 & 1
+        \end{pmatrix}
+
+    Converting to FC notation:
+
+    >>> import numpy as np
+    >>> from toqito.helper import cg_to_fc
+    >>> chsh_cg = np.array([[0, 0, 0], [0, 1, -1], [0, -1, 1]])
+    >>> cg_to_fc(chsh_cg)
+    array([[ 0.  ,  0.  ,  0.  ],
+           [ 0.  ,  0.25, -0.25],
+           [ 0.  , -0.25,  0.25]])
+
+    Consider a behaviour (probability distribution) in CG notation:
+
+    .. math::
+        P_{CG} =
+        \begin{pmatrix}
+            1 & 0.5 & 0.5 \\
+            0.5 & 0.25 & 0.25 \\
+            0.5 & 0.25 & 0.25
+        \end{pmatrix}
+
+    Converting to FC notation:
+
+    >>> import numpy as np
+    >>> from toqito.helper import cg_to_fc
+    >>> p_cg = np.array([[1, 0.5, 0.5], [0.5, 0.25, 0.25], [0.5, 0.25, 0.25]])
+    >>> cg_to_fc(p_cg, behaviour=True)
+    array([[1., 0., 0.],
+           [0., 0., 0.],
+           [0., 0., 0.]])
+
+    :param cg_mat: The matrix in Collins-Gisin notation.
+    :param behaviour: If True, assume input is a behaviour (default: False, assume functional).
+    :return: The matrix in Full Correlator notation.
+
+    .. note::
+        This function is adapted from the QETLAB MATLAB package function `CG2FC`.
+
+    References
+    ==========
+    .. bibliography::
+        :filter: docname in docnames
```

Comment:
You don't need this section if you are not adding a reference. But it might be better to add a reference as well as cite QETLAB. Take a look at the bib file in `docs/`.